elasticsearch/aws_plugin.sls

```diff
@@ -2,8 +2,9 @@ install_aws:
   cmd:
   - run
   - user: root
-  - name: '/usr/share/elasticsearch/bin/plugin install cloud-aws'
+  - name: '/usr/share/elasticsearch/bin/plugin install -b cloud-aws ; /sbin/service elasticsearch restart'
```

Comment:
the batch flag answers "yes" to the security question at the end of the install.